### ISSUE TITLE
Automatically trigger backup

### ISSUE THREAD

**Issue by quazar-omega** [anandnet/Harmony-Music#344]
**Is your feature request related to a problem or new idea? Please describe.**
Last time I updated the app I lost the history of played songs I had in my playlists, I had performed a backup only once to test it out figuring I would only actually use it for a device/system migration, but I was left lost data without messing with the app this time.

**Describe the solution you'd like**
This wouldn't be a problem if a backup could be automatically run daily (or after closing the app, or whichever other trigger). Then I would have been able to restore immediately after updating.

**Describe alternatives you've considered**
Religiously backing up manually every time I've used the app or before performing an update, latter is actually a problem since my updates come in automatically, so it's not really applicable.

**Additional context**
This has been already brought up in #16, but was closed by the author. I still believe it's a rather important feature, so I'd like to propose it again.

**Comment by quazar-omega** [comment]
I just realised that the recently played playlist just keeps 30 entries, so I probably made the wrong association there. I guess that's a separate issue to talk about.  
Still, I would find it useful to have automatic backups
